com.unity.cinemachine/Runtime/Behaviours/CinemachineFreeLookModifier.cs

```diff
@@ -20,6 +20,11 @@ public interface IModifiablePositionDamping
     {
         Vector3 PositionDamping { get; set; }
     }
+    
+    public interface IModifiableScreenPosition
+    {
+        (float, float) Screen { get; set; }
```

Comment:
Vector2 would be more straightforward, I think

Reply:
I used (float, float) because noise uses it to be consistent, but Vector2 is ok for me :) 
`(float, float) m_CenterNoise;`